lib/src/cardano_ledger_application.dart

```diff
@@ -29,7 +31,7 @@ import 'package:ledger_flutter/ledger_flutter.dart';
 /// https://github.com/cardano-foundation/ledger-app-cardano/blob/master/doc/design_doc.md
 /// https://github.com/cardano-foundation/ledgerjs-hw-app-cardano
 class CardanoLedgerApp {
-  static bool debugPrintEnabled = false;
+  static bool debugPrintEnabled = true;
```

Comment:
This should stay as `false` by default in here. It's not `final` on purpose. You can change it inside `example/lib/main.dart` when the app starts to be true when app is running in debug mode.

Reply:
done